app/Console/Command/ChangePlaceTypeOnPlacesWithEventEventType.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemIdentifier;
+use CultuurNet\UDB3\Offer\Commands\UpdateType;
+use CultuurNet\UDB3\Place\Commands\Moderation\Reject;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\StringLiteral;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+final class ChangePlaceTypeOnPlacesWithEventEventType extends AbstractCommand
+{
+    private SearchServiceInterface $searchService;
+
+    public function __construct(
+        CommandBus $commandBus,
+        SearchServiceInterface $searchService
+    ) {
+        $this->searchService = $searchService;
+        parent::__construct($commandBus);
+    }
+
+    public function configure(): void
+    {
+        $this->setName('place:eventtype:update:reject');
+
+        $this->setDescription(
+            'Updates all places with an Event eventType and optionally reject them'
+        );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->setVerbosity(OutputInterface::VERBOSITY_VERY_VERBOSE);
+        $logger = new ConsoleLogger($output);
+
+        $eventEventType = $this->askEventEventType($input, $output);
+        $placeEventType = $this->askPlaceEventType($input, $output);
+
+        $resultsGenerator = new ResultsGenerator(
+            $this->searchService,
+            ['created' => 'asc'],
+            100
+        );
+
+        $query = 'terms.id:' . $eventEventType;
+        if (!$this->askConfirmation($input, $output, $resultsGenerator->count($query))) {
+            return 0;
+        }
+
+        $reject = $this->askToReject($input, $output);
```

Comment:
Should this not be an option of the console command instead of a user question?

Reply:
In think if you make it an option, it could easily be forgotten. And once the type has been changed, it would be a lot harder to find those places which had an `EventType` for `Events`.
So I think it would be wiser to explicitly asked the user.

Reply:
I think the rejecting should be a different command

Reply:
> I think the rejecting should be a different command, according to the ticket:

Will that not make it tricky to keep track of the modified places?

Reply:
True, if we give them a new eventtype first. But I'm now wondering why that is even necessary in the case of places that should be rejected. Feels like trying to clean up data that does not matter anyway 🤔 (And it's not like a place that's about a concert is suddenly a semantically correct place just by getting the type "Zaal of expohal" - which is also why we reject them)

Anyway, we can keep it 1 command but if we go the route of not making it flexible at all I would also not ask this at all but infer it from the info in the ticket.

Reply:
Clarified in the ticket: We will change the eventtype but not update the workflowStatus